docs/learn/learn-nft.md

```diff
@@ -1,102 +1,108 @@
+
+NFT transfers can be decomposed into two independently transferable components:
+
+- core asset properties, consisting of identity and ownership data, and
+- optional metadata attributes.
+
+In certain cross-chain operations, such as staking, complete NFT teleportation is unnecessary. This was demonstrated through a successful [XCM implementation between Unique Network and Acala](https://unique.network/blog/unique-network-cross-chain-nft-proof-of-concept-is-here/), where a derivative NFT was created on the Acala chain representing the original NFT from Unique Network. The derivative implementation required only core asset properties, as metadata transfer was nonessential for staking and collateral use cases where only identity and ownership verification are required.
 
-Bridging to and from Substrate chains and EVM chains takes much effort but is a highly desired
-feature in the NFT industry. Merging the collector and customer base has significant implications,
-so multiple projects focus on making this possible.
+On the other hand, teleportation, i.e. a full transfer of identity ownership and metadata may be the optimal solution in the case of a full asset migration and the only available option for bridging mechanisms.
 
-Apart from RMRK (Substrate-to-Substrate seamless teleportation natively with [XCMP](learn-xcm.md))
-and Efinity (Paratoken), the following efforts are underway:
+Bridging to and from Substrate chains and EVM chains takes much effort but is a highly desired feature in the NFT industry. Merging the collector and customer base has significant implications, so multiple projects focus on making this possible.
 
-- [**MyNFT**](https://mynft.com/): an EVM to EVM bridging effort.
-- **RMRK \<-\> EVM** Simplification bridge: a bridge developed during the
-  [RMRK hackathon](https://rmrk.devpost.com) for porting RMRK NFTs into simplified IOUs on EVM
-  chains
+## What’s Next From Here?
 
-## References
+Utility NFTs are at a very early adoption stage.
 
-- [1]: [Investopedia](https://www.investopedia.com/terms/l/liars-poker.asp)
-- [2]:
-  [Unique Network's Chelobrick](https://unique.network/blog/chelobricks-making-waves-with-10-000-substrate-based-nfts/)
+If you consider an NFT as an ownable, universal, digital encapsulation of data, NFTs have a role as significant as that of a smart contract. Real-world asset tokenization, tradable digital coupons, virtual digital keys and badges, bundles of utility, resource and accomplishment tokens, data wrappers…  a likely advent of the era of discovery lays ahead.
```

Comment:
```suggestion
Utility NFTs are at a very early adoption stage.

If you consider an NFT an ownable, universal digital encapsulation of data, it has a role as significant as that of a smart contract. Real-world asset tokenization, tradable digital coupons, virtual digital keys and badges, bundles of utility, resource, and accomplishment tokens, data wrappers…  a likely advent of the era of discovery lays ahead.
```

Reply:
I get 'Applying suggestions on deleted lines is currently not supported' on a commit (?)